index.bs

```diff
@@ -447,6 +466,17 @@ To <dfn>parse an attribution destination</dfn> from a string |str|:
 1. Return the result of [=obtain a site|obtaining a site=] from |url|'s
     [=url/origin=].
 
+<h3 algorithm id="obtaining-random-attribution-mode">Obtaining a random attribution mode</h3>
+
+To <dfn>obtain a random attribution mode</dfn> given a positive integer |triggerDataCardinality|
+and a double |randomPickRate|:
+
+1. Let |possibleValues| be the [=set=] of [=attribution modes=] « (`"never"`, null) ».
```

Comment:
Consider pulling this into a separate algorithm, as it may get more complicated if more parameters are added in addition to the trigger data.

Reply:
Seems like we could wait to do that in a followup if it's needed.

Reply:
SGTM